package.json

```diff
@@ -20,7 +20,8 @@
     "reject": "0.0.1",
     "segmentio-integration": "^5.0.1",
     "to-title-case": "^0.1.5",
-    "ua-parser-js": "git+https://github.com/amplitude/ua-parser-js.git#fbdb1dc"
+    "ua-parser-js": "git+https://github.com/amplitude/ua-parser-js.git#fbdb1dc",
```

Comment:
Not sure, that was there when I pulled the integration in locally. I didn't change anything here, just added urlencode below.